go.mod

```diff
@@ -219,14 +219,14 @@ require (
 	go.uber.org/multierr v1.11.0 // indirect
 	golang.org/x/exp v0.0.0-20240719175910-8a7402abbf56 // indirect
 	golang.org/x/mod v0.21.0 // indirect
-	golang.org/x/text v0.19.0 // indirect
+	golang.org/x/text v0.23.0 // indirect
```

Comment:
The `unit` failure is caused by this bump, which is done because of the openshift/* bumps done above.

Technically, openshift/* bumps should be done in their own commit.